src/Shared/Data/Database/Skills.cs

```diff
@@ -124,9 +124,9 @@ protected override void ReadEntry(JObject entry)
 			info.UseType = entry.ReadEnum<SkillUseType>("useType");
 			info.Attribute = entry.ReadEnum<SkillAttribute>("attribute");
 
-			info.OverHeat = entry.ReadInt("overheat", 0);
-			info.OverHeatDelay = entry.ReadInt("overheatDelay", 0);
-			info.OverHeatGroup = entry.ReadString("overheatGroup", null);
+			info.OverHeat = entry.ReadInt("overheat");
```

Comment:
The overheat fields are values not every skill has. This is communicated by using explicit default values for when the fields are missing. Removing them takes that away from the code.

Reply:
But the defaults handle that already, I guess you'd like to leave it more verbose. If we were setting different default values, it'd make more sense.
`internal static int ReadInt(this JObject obj, string key, int def = 0) { return (int)(obj[key] ?? def); }`
`internal static string ReadString(this JObject obj, string key, string def = "") { return (string)(obj[key] ?? def); }`

Reply:
You're right, functionally it makes absolutey no difference. The only difference is that, when reading the code, you can see that default values are used, which communicates that these fields are optional in the data, and aren't necessarily set to something. It's not all that important, I just wanted to let you know why those defaults were set.

Reply:
Sounds good, will leave it in.